spark/interpreter/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -93,7 +93,8 @@ public void open() throws InterpreterException {
         }
       }
       // use local mode for embedded spark mode when spark.master is not found
-      conf.setIfMissing("spark.master", "local");
```

Comment:
I think this is the line of code that cause the issue in the mail-list

Reply:
yep, looks like...